src/test/java/org/gridsuite/study/server/SensitivityAnalysisTest.java

```diff
@@ -232,13 +237,27 @@ public MockResponse dispatch(RecordedRequest request) {
                            || path.matches("/v1/results/" + SENSITIVITY_ANALYSIS_OTHER_NODE_RESULT_UUID + "/status")) {
                     return new MockResponse().setResponseCode(200).setBody(SENSITIVITY_ANALYSIS_STATUS_JSON)
                         .addHeader("Content-Type", "application/json; charset=utf-8");
+                } else if (path.matches("/v1/results/" + SENSITIVITY_ANALYSIS_RESULT_UUID + "/tabbed\\?.*")
+                    || path.matches("/v1/results/" + SENSITIVITY_ANALYSIS_OTHER_NODE_RESULT_UUID + "/tabbed\\?.*")) {
+                    if (difficultyOfFind == 0) {
```

Comment:
strange way to mock server calls with external difficultyOfFind variable ...
better to use specific request path variables or specific request parameters ?

Reply:
Strange ? May be.
Unusual ? Yes for sure. But I found this way more compact and less prone to error than juggling with dummy IDs.
May be with an enum in stead of of plain integer, it would be less harsh ?